### DISCUSSION TITLE
Spanish translation

### DISCUSSION THREAD

**Discussion by johnfelipe** [pietrop/digital-paper-edit-electron#67] [language]
Can I contribute with spanish translation?

**Comment by johnfelipe** [comment]
if u send me strings in english can i share spanish translation for UI

**Comment by pietrop** [comment]
Thanks @johnfelipe 
Changing the UI to support multiple language also requires adding some kind of React library to handle that in [pietrop/digital-paper-edit-client](https://github.com/pietrop/digital-paper-edit-client). Do you know of any good once?

And would you be able to help with a PR?

_At the moment I don't have a lot of time for this. As I am working on more structural changes in [pietrop/slate-transcript-editor](https://github.com/pietrop/slate-transcript-editor) and [pietrop/digital-paper-edit-firebase](https://github.com/pietrop/digital-paper-edit-firebase) to make those more robust, and hopefully bring some of those back into [pietrop/digital-paper-edit-electron](https://github.com/pietrop/digital-paper-edit-electron)  [autoEdit3](https://www.autoedit.io)_
